tests/test_image_to_text_example.py

```diff
@@ -23,6 +23,8 @@
             ("HuggingFaceM4/idefics2-8b", 1, 21.89944593215077),
             ("meta-llama/Llama-3.2-11B-Vision-Instruct", 1, 18.974541922240313),
             ("tiiuae/falcon-11B-vlm", 1, 23.69260849957278),
+            ("Qwen/Qwen2-VL-2B-Instruct", 1, 13.319195250861407),
+            ("Qwen/Qwen2-VL-7B-Instruct", 1, 7.491897460793705),
```

Comment:
can we just add 1 test ?

Reply:
Here is 1 test for 2B and the other test for 7B model, which test do you think we should keep?

Reply:
7B

Reply:
Fixed.